yoti-sdk-api/src/main/java/com/yoti/api/client/Anchor.java

```diff
@@ -10,23 +10,27 @@
 public interface Anchor {
 
     /**
-     * The type of Anchor - most likely either SOURCE or VERIFIER, but it's possible that new Anchor types will be added in future
-     *
+     * The type of Anchor - most likely either but it's possible that new Anchor types will be added in future
+     * "SOURCE" or "VERIFIER"
```

Comment:
Should `"SOURCE" or "VERIFIER"` be after the sentence? Not sure that `most likely either but` reads well?